packages/documentation/src/stories/packages/components-react/react-app-module.sample.ts

```diff
@@ -0,0 +1,11 @@
+// app.tsx
+
+import { PostComponent } from '@swisspost/design-system-components-react';
+
+function App() {
+  return (
+    <>
+      <PostComponent>My component content</PostComponent>
+    </>
+  );
+}
```

Comment:
You can use the post-icon component as an example as we have in the components for Angular docs:
```suggestion
import { PostIcon } from '@swisspost/design-system-components-react';

function App() {
  return (
    <PostIcon name="iconName">My component content</PostIcon>
  );
}
```